script/setup/install-protobuf

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# Downloads and installs protobuf
```

Comment:
There are a few lines in `.travis.yml` that could be replaced by `./script/setup/install-protobuf`